code/modules/shuttle/shuttle.dm

```diff
@@ -821,3 +843,8 @@
 
 /obj/docking_port/mobile/emergency/on_emergency_dock()
 	return
+
+//This can go here, fuck it
```

Comment:
no

Reply:
where should it go then

Reply:
You could either give it its own file (which is a little excessive) or make use of the file for the ripple effect as a more general use shuttle effects file